src/integration/pages/frontPage/serbian/__snapshots__/amp.test.js.snap

```diff
@@ -117,27 +117,16 @@ exports[`AMP Front Page Analytics ATI 1`] = `
 exports[`AMP Front Page Analytics chartbeat 1`] = `
 {
   "vars": {
-    "contentType": "Index",
+    "contentType": "home",
     "domain": "test.bbc.co.uk",
-    "sections": "Serbian, Serbian - IDX",
-    "title": "Početna strana - BBC News na srpskom",
+    "sections": "Serbian, Serbian - home",
+    "title": "Početna stranica",
     "uid": 50924,
     "virtualReferrer": "\${documentReferrer}",
   },
 }
 `;
 
-exports[`AMP Front Page Analytics comscore 1`] = `
```

Comment:
This is OK - Comscore was a legacy analytics tracker, and is no longer present on homepages